crates/tombi-wasm-compat/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tombi-wasm-compat"
```

Comment:
Given that this crate is only about the Send feature of features, a crate name of `tombi-features` would be better; a features name similar to tokio would make it easier to understand the intent.

Split the features of this crate into `tombi-features` and `tombi-url`.

Reply:
Yes, that indeed seems better (and I'm assuming you meant "future" as a typo there).

Reply:
I meant in the future 😅